docs/development_environment.md

```diff
@@ -11,13 +11,13 @@ You'll need to set up a development environment if you want to develop a new fea
 Install the core dependencies.
 
 ```bash
-$ sudo apt-get install python3-pip python3-dev python3-venv
```

Comment:
Not necessarily related to this PR, but would it be better if we could somehow show the `bash` part in the UI? This would allow double-clicking it for copy selection (for accessibility & usability).

Reply:
@balloob Added back the bash and powershell-symbols: https://github.com/home-assistant/developers.home-assistant/pull/173/commits/b1a3bf0ec19c4a7c0290011a6b6e81fb7b3d7bdd